src/main/java/cn/nukkit/block/BlockGrassPath.java

```diff
@@ -59,6 +64,13 @@ public boolean canSilkTouch() {
 
     @Override
     public int onUpdate(int type) {
+        if (type == Level.BLOCK_UPDATE_RANDOM) {
+            if (this.up().isSolid()) {
+                this.level.setBlock(this, Block.get(BlockID.DIRT), false, true);
+            }
+            
+            return Level.BLOCK_UPDATE_RANDOM;
+        }
```

Comment:
According to single-player tests, It should decay immediately after a solid block is placed on top of it, so it should be `BLOCK_UPDATE_NORMAL`

Reply:
I think I accidentally write it, I edited to BLOCK_UPDATE_NORMAL